firewall/firewall_linux.go

```diff
@@ -58,7 +59,17 @@ func getInterfaceName(dst net.IPNet) (string, error) {
 	}
 	routes, err := h.RouteGet(dst.IP)
 	if err != nil {
-		return "", err
+		if !strings.Contains(err.Error(), "network is unreachable") {
+			return "", err
+		}
+		//if ipv4 address is unreachable, try ipv6 address
+		dst.IP = net.ParseIP("2606:4700:4700::1111")
```

Comment:
why is this address hard coded?

Reply:
Good point @abhishek9686 .  2606:4700:4700::1111 is the ipv6 format for 1.1.1.1.  
I added a commit and changed it to constant.  Please have a review again. Thanks.